.github/chainguard/Reminder.sts.yaml

```diff
@@ -0,0 +1,82 @@
+issuer: https://token.actions.githubusercontent.com
+subject_pattern: repo:shopware/shopware:.*
+claim_pattern:
+  workflow_ref: shopware/shopware/.github/workflows/.*
+
+permissions:
+  metadata: read
+  members: read
+  pull_requests: write
+
+# All repositories of the shopware organization
+#
+# $: gh repo list shopware --limit=999 --visibility="public" --json="name" --jq=".[] |= .name" | yq -p json '{ "repositories": . }'
+repositories:
```

Comment:
The reminder workflow[^1] would need to be able to close Pull Requests across our whole organization.

According to octo-sts docs[^2], for org level policies, we need to define the whole list of repositories to grant access to.
As of now, I've only added public ones, we'd need to add private ones as well.

WDYT?

[^1]: https://github.com/shopware/shopware/pull/10748/files
[^2]: https://github.com/octo-sts/app?tab=readme-ov-file#the-trust-policy